apps/stats/test/utils/hook-testing-utils.ts

```
@@ -0,0 +1,314 @@
+    input: TInput,
+    output: TOutput,
+    transformFunction: (input: TInput) => TOutput
+) => {
+    const result = transformFunction(input);
+    expect(result).toEqual(output);
+};
+
+/**
+ * Utility for testing memoization
+ */
+export const expectMemoization = (hookFunction: Function, deps: any[]) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)